docs/guides/verify-shapella-upgrade-manual.md

```diff
@@ -0,0 +1,530 @@
+# Shapella mainnet parameters
+
+## Mainnet addresses
+
+```python
+lidoLocator = 0xC1d0b3DE6792Bf6b4b37EccdcC24e45978Cfd2Eb
+legacyOracle = 0x442af784A788A5bd6F42A01Ebe9F287a871243fb  # was `LidoOracle` before
+
+# See https://docs.lido.fi/deployed-contracts/
+# Old implementation was `Withdrawals Manager Stub`
+withdrawalsVault = 0xB9D7934878B5FB9610B3fE8A5e441e8fad7E293f
+
+# Same as withdrawalsVault with 0x01 type prefix
+# also see https://etherscan.io/address/0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84#readProxyContract#F23
+withdrawalsCredentials = 0x010000000000000000000000b9d7934878b5fb9610b3fe8a5e441e8fad7e293f
+```
+
+## NodeOperatorsRegistry
+Single staking module named "Curated" added to StakingRouter.
+
+```python
+# See https://snapshot.org/#/lido-snapshot.eth/proposal/0xa4eb1220a15d46a1825d5a0f44de1b34644d4aa6bb95f910b86b29bb7654e330
```

Comment:
I'd suggest to quote: 'Their status shall revert to “In good standing” after **5 days** (i.e. provided any newly received validator exit requests are processed timely)' 

The same is for other params